docs/pages/includes/install-linux.mdx

```diff
@@ -1,220 +1,186 @@
+   | sudo tee /etc/apt/sources.list.d/teleport.list > /dev/null
+
+  $ sudo apt-get update
+  $ sudo apt-get install teleport
+  ```
 
-  <TabItem label="ARM64/ARMv8 (64-bit)">
-        ```code
-        $ curl https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz.sha256
-        # <checksum> <filename>
-        $ curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
-        $ shasum -a 256 teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
-        # Verify that the checksums match
-        $ tar -xzf teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
-        $ cd teleport
-        $ sudo ./install
-        ```
   </TabItem>
+  <TabItem label="Amazon Linux 2/RHEL (RPM)">
+
+  ```code
+  # Source variables about OS version
+  $ source /etc/os-release
+  # Add the Teleport YUM repository for v(=teleport.major_version=). You'll need to update this
+  # file for each major release of Teleport.
+  # Note: if using a fork of RHEL/CentOS or Amazon Linux you may need to use '$ID_LIKE'
+  # and the codename your distro was forked from instead of '$ID'
+  # Supported versions are listed here: https://github.com/gravitational/teleport/blob/master/build.assets/tooling/cmd/build-os-package-repos/runners.go#L133-L153
+  $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+  $ sudo dnf install teleport
+  #
+  # Tip: Add /usr/local/bin to path used by sudo (so 'sudo tctl users add' will work as per the docs)
+  # echo "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin" > /etc/sudoers.d/secure_path
+  #
+  # Optional:  Using YUM on older distributions
+  #$ sudo yum-config-manager --add-repo $(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/v(=teleport.major_version=)/teleport.repo")
+  #$ sudo yum install teleport
+  ```
 
-  <TabItem label="Debian/Ubuntu Legacy (DEB)">
-        ```code
-        # Using this APT repo may result in breaking upgrades upon "apt upgrade" as all major versions will be
-        # published under the same component. We recommend following the instructions in the
-        # "Debian/Ubuntu (DEB)" tab instead.
-        # Download Teleport's PGP public key
-        $ sudo curl https://deb.releases.teleport.dev/teleport-pubkey.asc \
-          -o /usr/share/keyrings/teleport-archive-keyring.asc
-        # Add the Teleport APT repository
-        $ echo "deb [signed-by=/usr/share/keyrings/teleport-archive-keyring.asc] https://deb.releases.teleport.dev/ stable main" \
-         | sudo tee /etc/apt/sources.list.d/teleport.list > /dev/null
-
-        $ sudo apt-get update
-        $ sudo apt-get install teleport
-        ```
   </TabItem>
+  <TabItem label="Tarball">
+
+  ```code
+  $ curl https://get.gravitational.com/teleport-v(=teleport.version=)-linux-<Var name="$SYSTEM-ARCH" description="The CPU architecture of the system Teleport will be installed on"/>-bin.tar.gz.sha256
```

Comment:
IIUC, get.gravitational.com is old infra (Houston) which is being replaced by cdn.teleport.dev. If the CDN is up now (I think it is but we should confirm) then maybe we should replace these links.

Reply:
Adjusted in [be612c5](https://github.com/gravitational/teleport/pull/17498/commits/be612c53e0dad4455b166832cba2b4b6e323578f)